src/utils/validation-schemas.ts

```diff
@@ -41,7 +41,13 @@ export const amountSchema = z
  */
 const baseAddressSchema = z
   .string()
-  .transform((val) => sanitizeAddress(val))
+  .transform((val) => {
+    try {
+      return sanitizeAddress(val);
+    } catch (error) {
+      return '';
+    }
+  })
```

Comment:
if we're using this pattern a lot, could you extract it?

```
export const sanitizeAddressOrEmpty = (address: string): string => {
  try {
    return sanitizeAddress(address);
  }
  catch (e) {
    return '';
  }
};
```

Reply:
Done, thank you for the 🦅 eyes 💯